tests/publiccloud/suspending.pm

```diff
@@ -0,0 +1,52 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: cloud-netconfig-{azure, ec2, gce}
+# Summary: This test ensures the consistency of cloud-netconfig's
+# functionality. The test shall be conducted on a VM in the cloud
+# infrastructure of a supported CSP.
+#
+# Test that a VM can be suspended and resumed without successfully
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use Test::Assert qw(assert_equals assert_not_equals);
+use testapi;
+use utils qw(script_retry);
+use version_utils qw(is_sle);
+use publiccloud::utils qw(is_gce);
+
+sub run {
+    my ($self, $args) = @_;
+    my $instance = $args->{my_instance};
+    my $provider = $args->{my_provider};
+
+    unless (is_gce()) {
+        record_info('unsupported', 'The hibernation is currently supported only on GCE.');
+        return 1;
+    }
+    if (is_gce() && is_sle('=15-SP7')) {
```

Comment:
```suggestion
    if (is_sle('=15-SP7')) {
```
you don't need to check for gce here , if it is not you simply not reach this line